babel/src/main/java/org/apache/calcite/sql/babel/SqlCreateAttributeIsolatedLoading.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to you under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.calcite.sql.babel;
+
+import org.apache.calcite.sql.SqlWriter;
+import org.apache.calcite.sql.parser.SqlParserPos;
+
+/**
+ * A <code>SqlCreateAttributeIsolatedLoading</code> is a CREATE TABLE option
+ * for the WITH ISOLATED LOADING attribute.
+ */
+public class SqlCreateAttributeIsolatedLoading extends SqlCreateAttribute {
+
+  private boolean no;
+  private boolean concurrent;
+  private OperationLevel operationLevel;
+
+  /**
+   * Creates a {@code SqlCreateAttributeIsolatedLoading}.
+   *
+   * @param no  Defines table as non-load isolated
```

Comment:
I would recommend something a little more descriptive than "no", maybe "nonLoadIsolated"

Reply:
Yeah no is pretty unclear, I'll change the name